src/index.html

```diff
@@ -10,23 +10,40 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
-    <title>Document</title>
+    <title>Search bar airbnb</title>
     <link
       rel="stylesheet"
-      href="style.css"
+      href="./style.css"
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="big-searchbar"
```

Comment:
There's a better way to name it, you already did it with the input, if you want I can describe it more but I think you can figure it out - by the way there's also a BEM problem there

Reply:
Indeed. I'll update them!